<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DzikBook</class>
 <widget class="QMainWindow" name="DzikBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DzikBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list_contacts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_messages">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>590</width>
      <height>340</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>590</width>
      <height>340</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>590</width>
      <height>340</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_new_message">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>591</width>
      <height>141</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_send_message">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;XDDDDDDDDDDDDDDDDDD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;HALOOOOO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Wyślij wiadomość</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_profile">
    <property name="title">
     <string>Profil</string>
    </property>
    <addaction name="action_change_pass"/>
    <addaction name="action_change_infos"/>
   </widget>
   <widget class="QMenu" name="menu_friends">
    <property name="title">
     <string>Znajomi</string>
    </property>
    <addaction name="action_friens_list"/>
    <addaction name="action_find_friends"/>
   </widget>
   <widget class="QMenu" name="menuKonwersacje">
    <property name="title">
     <string>Konwersacje</string>
    </property>
   </widget>
   <addaction name="menu_profile"/>
   <addaction name="menu_friends"/>
   <addaction name="menuKonwersacje"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_change_pass">
   <property name="text">
    <string>Zmiana hasła</string>
   </property>
  </action>
  <action name="action_change_infos">
   <property name="text">
    <string>Zmiana danych</string>
   </property>
  </action>
  <action name="action_friens_list">
   <property name="text">
    <string>Lista znajomych</string>
   </property>
  </action>
  <action name="action_find_friends">
   <property name="text">
    <string>Szukaj znajomych</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
